index.js

```diff
@@ -149,8 +151,8 @@ class Pool extends EventEmitter {
 
   _remove (client) {
     const removed = removeWhere(
-      this._idle,
```

Comment:
This change shouldn’t be here.

Reply:
I don't remember changing anything here, but if I did I will roll it back.